lists/list.css

```diff
@@ -0,0 +1,132 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root{
+    --primary-color: #34D399;
+}
+
+
+body{
+    margin: 0 auto;
+    font-family: 'Fredoka', sans-serif;
+}
+
+#tweet-img{
+    width:30px;
+}
+
+.line{
+    margin: 1rem;
+    padding: 0.1px;
+    background-color: gray;
+}
+
+iframe{
+    margin-top: 1rem;
+    min-width: 90%;
+}
+
+.desc-txt{
+    
+    margin-top: 0.5rem;
+    font-weight: 600;
+}
+
+.container{
+    position: relative;
+}
+.sidenav-container{
+    position: fixed;
+}
+
+.main-container{
+    position: absolute;
+    left: 10rem;
+}
+
+.navigation{
+    background-color: var(--primary-color);
+    color: white;
+    padding: 0.1rem;
+    border-bottom-right-radius: 1rem;
+}
+
+.navigation .nav-brand{
+    font-weight: bold;
+    font-size: 2rem;
+    
+}
+
+.nav-pills{
+    text-align: right;
+}
+.list-non-bullet{
+    list-style: none;
+}
+
+.inline-item{
+    display:inline;
+    padding: 0rem 0.5rem;
+}
+
+.heading-text{
+    font-size: 1.3rem;
+    font-weight: bold;
+}
+.links{
+    text-decoration: none;
+    color: black;
+    font-size: 1.5rem;
+}
+.links:hover{
+    color: var(--primary-color);
+}
+.side-nav-link{
+    margin-top: 1rem;
+}
+
+
+.list-container{
+    display: flex;
+    flex-direction: row;
```

Comment:
this is a default flex property. No need to specify. You can read about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#:~:text=Understanding%20WCAG%202.0-,Formal%20definition,-Initial%20value).